### PR TITLE
AES TA: Fix compiler errors on AArch64

### PR DESCRIPTION
**Pull request by jbech-linaro**
This fixes issues seen on Travis during the weekend:
https://travis-ci.org/OP-TEE/build/jobs/301148288#L4676-L4705

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>

### CODE REVIEW

File: aes/ta/aes_ta.c [hunk 1]
```diff
@@ -247,8 +247,8 @@ static TEE_Result set_aes_key(void *session, uint32_t param_types,
 	key_sz = params[0].memref.size;
 
 	if (key_sz != sess->key_size) {
-		EMSG("Worng key size %d, expect %d bytes",
-				key_sz, sess->key_size);
+		EMSG("Wrong key size %d, expect %d bytes",
```
Comment (etienne-lms) [inline aes/ta/aes_ta.c]: `%d` is okay for an unsigned ?

Comment (jbech-linaro) [reply aes/ta/aes_ta.c]: `%lu`? `%zu`?

Comment (jbech-linaro) [reply aes/ta/aes_ta.c]: Or even the `PRIu32`?

Comment (jenswi-linaro) [reply aes/ta/aes_ta.c]: Given that `key_sz` and `sess->key_size` is changed to a `uint32_t` you should use `% PRIu32`. Before with `size_t` `%zu` would a have been appropriate.

File: aes/ta/aes_ta.c [hunk 2]
```diff
@@ -247,8 +248,8 @@ static TEE_Result set_aes_key(void *session, uint32_t param_types,
 	key_sz = params[0].memref.size;
 
 	if (key_sz != sess->key_size) {
-		EMSG("Worng key size %d, expect %d bytes",
-				key_sz, sess->key_size);
+		EMSG("Wrong key size %" PRIu32 ", expect %d bytes",
```
Comment (etienne-lms) [inline aes/ta/aes_ta.c]: and also for the 2nd `%d` => `" PRIxu32 "`.

Comment (jbech-linaro) [reply aes/ta/aes_ta.c]: Of course!

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Comment by jbech-linaro** [comment]
Squashed, rebased and tag(s) applied (should be) ready for merge! Thanks!

**Pull request merged**